src/components/views/settings/SpellCheckSettings.tsx

```diff
@@ -87,8 +90,9 @@ export default class SpellCheckLanguages extends React.Component<SpellCheckLangu
     };
 
     public render(): React.ReactNode {
+        const intl = new Intl.DisplayNames([getUserLanguage()], { type: "language", style: "short" });
```

Comment:
does `style: "short"` actually do anything here, ooi?

If not, I'd favour omitting it.

Reply:
Yes.

![image](https://github.com/matrix-org/matrix-react-sdk/assets/2403652/e0feeeee-553a-4789-b8d5-db10f98c2ae4)

The default is `long`.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DisplayNames/DisplayNames#options